controllers/kserve_customcacert_controller_test.go

```diff
@@ -78,21 +77,6 @@ var _ = Describe("KServe Custom CA Cert ConfigMap controller", func() {
 	})
 })
 
-func waitForConfigMap(cli client.Client, namespace, configmapName string, timeout time.Duration) (*corev1.ConfigMap, error) {
```

Comment:
What was the reason to move this to another file?

Reply:
earlier it was only used in this file but now it is a common function used in multiple places so I moved it to the test suite file